docs/pages/access-controls/access-requests/access-request-configuration.mdx

```diff
@@ -0,0 +1,622 @@
+  `db-writer-us-west-2` roles.
+
+To add the values of `claims_to_roles` to the lists of role matchers, the Auth
+Service evalutes template expressions with the user's traits. See the [Role
+Templates](../guides/role-templates.mdx#templating-in-access-reqeusts) for more
+information on how Teleport executes template expressions with user traits in
+the `claims_to_roles` field.
+
+In the `employee` role above, if the user is in the `admins` group (as declared
+by their single sign-on provider), this role allows them to request access to
+all roles.  If they are in the `contractors` group, this role denies them access
+to request any roles.
+
+### Restrict resource requests
+
+Users can also submit Access Requests for a specific Teleport resource.
+
+The following fields in a Teleport role indicate which roles a user assumes when
+they search for a Teleport resource:
+
+- `allow.request.search_as_roles`
+- `deny.request.search_as_roles`
+
+For example, the following role enables a user to search for resources by
+assuming all roles _except_ for the `k8s-viewer` role:
+
+```yaml
+# requester.yaml
+kind: role
+version: v6
+metadata:
+  name: k8s-denier
+spec:
+  allow:
+    request:
+      search_as_roles:
+        - '*'
+ deny:
+   request:
+      search_as_roles:
+        - k8s-viewer
+```
+
+As with [configuring role requests](#restrict-role-requests), the
+`request.search_as_roles` field is a list of role matchers that can include
+literal role names, wildcards, and regular expressions.
+
+The Teleport Auth Service combines the values of these fields for all of a
+user's Teleport roles in order to validate the user's Access Requests. 
+
+When a user attempts to list Teleport resources (such as servers and databases)
+or Kubernetes resources (such as pods and deployments), the Auth Service checks
+the roles that the user is allowed to search as. If a user's Teleport roles _or_
+a role specified in `search_as_roles` allow the user to search for a resource,
+the Teleport Auth Service will return information about the resource.
+
+When a user requests access to a resource ID, the Teleport Auth Service does the
+following:
+
+1. Collects all the roles named in `allow.request.search_as_roles`, filtering
+   these to exclude roles specified in `deny.request.search_as_roles` or
+   `deny.request.roles`. 
+1. Determines which of the remaining roles can access the requested resource.
+   For a Resource Access Request to be valid, one of the role matchers listed in
+   a user's `search_as_roles` configuration must match a role that permits
+   access to the requested resources.
+
+## How long access lasts
+
+You can configure the length of time that Teleport will grant a user elevated
+privileges for an approved Access Request.
+
+### The `request.max_duration` field
+
+The `max_duration` option indicates the maximum length of time that a user is
+allowed to have elevated privileges for particular roles. You can specify it
```

Comment:
I would consider mentioning `max_duration` towards the end, since it's basically an exception to the default rules.

I might also clarify that a `max_duration` of `4d` doesn't mean you can have continuous access for 4 days - you're still limited by the session TTL, but you can re-assume the access to restart the session TTL clock for up to 4 days.